tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```diff
@@ -1,9 +1,9 @@
 ﻿Languages:
   Net:
     DefaultVersions:
-    - netcoreapp3.1
-    OptionalVersions:
     - net6.0
+    OptionalVersions:
+    - net7.0
```

Comment:
This requires a review by @mikeharder

Reply:
These are the versions used to run perf tests so these shouldn't be changed now.  I have a separate work item to update language versions for all perf tests to latest.